docs/api.md

```
@@ -8,13 +8,24 @@
         filters:
             - "!__init__"
             - "!_metadata"
-
-::: holm.ErrorHandlerMapping
+            
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 12

(MD009, no-trailing-spaces)